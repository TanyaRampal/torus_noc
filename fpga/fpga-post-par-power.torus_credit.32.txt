Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
---------------------------------------------------------------------------------------------
| Tool Version     : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date             : Fri Dec  6 14:42:53 2024
| Host             : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command          : report_power -file fpga-post-par-power.torus_credit.32.txt
| Design           : torus_credit
| Device           : xc7z020clg400-1
| Design State     : routed
| Grade            : commercial
| Process          : typical
| Characterization : Production
---------------------------------------------------------------------------------------------

Power Report

Table of Contents
-----------------
1. Summary
1.1 On-Chip Components
1.2 Power Supply Summary
1.3 Confidence Level
2. Settings
2.1 Environment
2.2 Clock Constraints
3. Detailed Reports
3.1 By Hierarchy

1. Summary
----------

+--------------------------+-------------------------+
| Total On-Chip Power (W)  | 2.644                   |
| Design Power Budget (W)  | Unspecified*            |
| Power Budget Margin (W)  | NA                      |
| Dynamic (W)              | 2.480                   |
| Device Static (W)        | 0.164                   |
| Effective TJA (C/W)      | 11.5                    |
| Max Ambient (C)          | 54.5                    |
| Junction Temperature (C) | 55.5                    |
| Confidence Level         | Medium                  |
| Setting File             | ---                     |
| Simulation Activity File | torus_credit_D_W32.saif |
| Design Nets Matched      | 1%   (92/9759)          |
+--------------------------+-------------------------+
* Specify Design Power Budget using, set_operating_conditions -design_power_budget <value in Watts>


1.1 On-Chip Components
----------------------

+--------------------------+-----------+----------+-----------+-----------------+
| On-Chip                  | Power (W) | Used     | Available | Utilization (%) |
+--------------------------+-----------+----------+-----------+-----------------+
| Clocks                   |     0.044 |        3 |       --- |             --- |
| Slice Logic              |     0.558 |     9806 |       --- |             --- |
|   LUT as Logic           |     0.491 |     3735 |     53200 |            7.02 |
|   Register               |     0.061 |     2810 |    106400 |            2.64 |
|   LUT as Distributed RAM |     0.007 |     1152 |     17400 |            6.62 |
|   CARRY4                 |    <0.001 |       32 |     13300 |            0.24 |
|   Others                 |     0.000 |      161 |       --- |             --- |
| Signals                  |     1.878 |     7548 |       --- |             --- |
| Static Power             |     0.164 |          |           |                 |
| Total                    |     2.644 |          |           |                 |
+--------------------------+-----------+----------+-----------+-----------------+


1.2 Power Supply Summary
------------------------

+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+
| Source    | Voltage (V) | Total (A) | Dynamic (A) | Static (A) | Powerup (A) | Budget (A)  | Margin (A) |
+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+
| Vccint    |       1.000 |     2.503 |       2.480 |      0.023 |       NA    | Unspecified | NA         |
| Vccaux    |       1.800 |     0.019 |       0.000 |      0.019 |       NA    | Unspecified | NA         |
| Vcco33    |       3.300 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco25    |       2.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco18    |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco15    |       1.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco135   |       1.350 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco12    |       1.200 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccaux_io |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccbram   |       1.000 |     0.001 |       0.000 |      0.001 |       NA    | Unspecified | NA         |
| MGTAVcc   |       1.000 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| MGTAVtt   |       1.200 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| MGTVccaux |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccpint   |       1.000 |     0.044 |       0.000 |      0.044 |       NA    | Unspecified | NA         |
| Vccpaux   |       1.800 |     0.010 |       0.000 |      0.010 |       NA    | Unspecified | NA         |
| Vccpll    |       1.800 |     0.003 |       0.000 |      0.003 |       NA    | Unspecified | NA         |
| Vcco_ddr  |       1.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco_mio0 |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco_mio1 |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccadc    |       1.800 |     0.020 |       0.000 |      0.020 |       NA    | Unspecified | NA         |
+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+


1.3 Confidence Level
--------------------

+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+
| User Input Data             | Confidence | Details                                        | Action                                                                                                     |
+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+
| Design implementation state | High       | Design is routed                               |                                                                                                            |
| Clock nodes activity        | High       | User specified more than 95% of clocks         |                                                                                                            |
| I/O nodes activity          | High       | User specified more than 95% of inputs         |                                                                                                            |
| Internal nodes activity     | Medium     | User specified less than 25% of internal nodes | Provide missing internal nodes activity with simulation results or by editing the "By Resource Type" views |
| Device models               | High       | Device models are Production                   |                                                                                                            |
|                             |            |                                                |                                                                                                            |
| Overall confidence level    | Medium     |                                                |                                                                                                            |
+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+


2. Settings
-----------

2.1 Environment
---------------

+-----------------------+------------------------+
| Ambient Temp (C)      | 25.0                   |
| ThetaJA (C/W)         | 11.5                   |
| Airflow (LFM)         | 250                    |
| Heat Sink             | none                   |
| ThetaSA (C/W)         | 0.0                    |
| Board Selection       | medium (10"x10")       |
| # of Board Layers     | 8to11 (8 to 11 Layers) |
| Board Temperature (C) | 25.0                   |
+-----------------------+------------------------+


2.2 Clock Constraints
---------------------

+-------+--------+-----------------+
| Clock | Domain | Constraint (ns) |
+-------+--------+-----------------+
| clk   | clk    |             6.0 |
+-------+--------+-----------------+


3. Detailed Reports
-------------------

3.1 By Hierarchy
----------------

+-------------------------------+-----------+
| Name                          | Power (W) |
+-------------------------------+-----------+
| torus_credit                  |     2.480 |
|   ys[0].xs[0].client_xy       |     0.010 |
|     regulator                 |     0.004 |
|   ys[0].xs[0].torus_switch_xy |     1.983 |
|     east_conn_tx              |     0.007 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[0].xs[1].client_xy       |     0.027 |
|     regulator                 |     0.020 |
|   ys[0].xs[1].torus_switch_xy |     0.016 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[0].xs[2].client_xy       |     0.031 |
|     regulator                 |     0.023 |
|   ys[0].xs[2].torus_switch_xy |     0.015 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[0].xs[3].client_xy       |     0.012 |
|   ys[0].xs[3].torus_switch_xy |     0.015 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.009 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.002 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[1].xs[0].client_xy       |     0.033 |
|     regulator                 |     0.025 |
|   ys[1].xs[0].torus_switch_xy |     0.014 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[1].xs[1].client_xy       |     0.005 |
|   ys[1].xs[1].torus_switch_xy |     0.011 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[1].xs[2].client_xy       |     0.035 |
|     regulator                 |     0.027 |
|   ys[1].xs[2].torus_switch_xy |     0.015 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[1].xs[3].client_xy       |     0.004 |
|   ys[1].xs[3].torus_switch_xy |     0.011 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[2].xs[0].client_xy       |     0.031 |
|     regulator                 |     0.022 |
|   ys[2].xs[0].torus_switch_xy |     0.016 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[2].xs[1].client_xy       |     0.006 |
|   ys[2].xs[1].torus_switch_xy |     0.012 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[2].xs[2].client_xy       |     0.035 |
|     regulator                 |     0.025 |
|   ys[2].xs[2].torus_switch_xy |     0.015 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[2].xs[3].client_xy       |     0.005 |
|   ys[2].xs[3].torus_switch_xy |     0.012 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.002 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
|   ys[3].xs[0].client_xy       |     0.024 |
|     regulator                 |     0.017 |
|   ys[3].xs[0].torus_switch_xy |     0.013 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[3].xs[1].client_xy       |     0.027 |
|     regulator                 |     0.020 |
|   ys[3].xs[1].torus_switch_xy |     0.014 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.007 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.004 |
|   ys[3].xs[2].client_xy       |     0.005 |
|   ys[3].xs[2].torus_switch_xy |     0.013 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.009 |
|       gen_vc_logic[0].vc_fifo |     0.004 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[3].xs[3].client_xy       |     0.003 |
|   ys[3].xs[3].torus_switch_xy |     0.012 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.008 |
|       gen_vc_logic[0].vc_fifo |     0.001 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.005 |
+-------------------------------+-----------+


