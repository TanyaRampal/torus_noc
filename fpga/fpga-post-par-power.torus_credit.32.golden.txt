Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
---------------------------------------------------------------------------------------------
| Tool Version     : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date             : Tue Nov 26 14:40:34 2024
| Host             : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command          : report_power -file fpga-post-par-power.torus_credit.32.txt
| Design           : torus_credit
| Device           : xc7z020clg400-1
| Design State     : routed
| Grade            : commercial
| Process          : typical
| Characterization : Production
---------------------------------------------------------------------------------------------

Power Report

Table of Contents
-----------------
1. Summary
1.1 On-Chip Components
1.2 Power Supply Summary
1.3 Confidence Level
2. Settings
2.1 Environment
2.2 Clock Constraints
3. Detailed Reports
3.1 By Hierarchy

1. Summary
----------

+--------------------------+----------------------------------+
| Total On-Chip Power (W)  | 33.414 (Junction temp exceeded!) |
| Design Power Budget (W)  | Unspecified*                     |
| Power Budget Margin (W)  | NA                               |
| Dynamic (W)              | 32.376                           |
| Device Static (W)        | 1.038                            |
| Effective TJA (C/W)      | 11.5                             |
| Max Ambient (C)          | 0.0                              |
| Junction Temperature (C) | 125.0                            |
| Confidence Level         | Medium                           |
| Setting File             | ---                              |
| Simulation Activity File | torus_credit_D_W32.saif          |
| Design Nets Matched      | 6%   (670/11449)                 |
+--------------------------+----------------------------------+
* Specify Design Power Budget using, set_operating_conditions -design_power_budget <value in Watts>


1.1 On-Chip Components
----------------------

+--------------------------+-----------+----------+-----------+-----------------+
| On-Chip                  | Power (W) | Used     | Available | Utilization (%) |
+--------------------------+-----------+----------+-----------+-----------------+
| Clocks                   |     0.028 |        3 |       --- |             --- |
| Slice Logic              |     4.270 |    11016 |       --- |             --- |
|   LUT as Logic           |     3.309 |     3639 |     53200 |            6.84 |
|   Register               |     0.955 |     3344 |    106400 |            3.14 |
|   LUT as Distributed RAM |     0.006 |     1152 |     17400 |            6.62 |
|   CARRY4                 |    <0.001 |      224 |     13300 |            1.68 |
|   F7/F8 Muxes            |    <0.001 |       16 |     53200 |            0.03 |
|   Others                 |     0.000 |      209 |       --- |             --- |
| Signals                  |    28.079 |     8271 |       --- |             --- |
| Static Power             |     1.038 |          |           |                 |
| Total                    |    33.414 |          |           |                 |
+--------------------------+-----------+----------+-----------+-----------------+


1.2 Power Supply Summary
------------------------

+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+
| Source    | Voltage (V) | Total (A) | Dynamic (A) | Static (A) | Powerup (A) | Budget (A)  | Margin (A) |
+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+
| Vccint    |       1.000 |    32.675 |      32.376 |      0.299 |       NA    | Unspecified | NA         |
| Vccaux    |       1.800 |     0.100 |       0.000 |      0.100 |       NA    | Unspecified | NA         |
| Vcco33    |       3.300 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco25    |       2.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco18    |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco15    |       1.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco135   |       1.350 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco12    |       1.200 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccaux_io |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccbram   |       1.000 |     0.026 |       0.000 |      0.026 |       NA    | Unspecified | NA         |
| MGTAVcc   |       1.000 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| MGTAVtt   |       1.200 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| MGTVccaux |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccpint   |       1.000 |     0.473 |       0.000 |      0.473 |       NA    | Unspecified | NA         |
| Vccpaux   |       1.800 |     0.010 |       0.000 |      0.010 |       NA    | Unspecified | NA         |
| Vccpll    |       1.800 |     0.003 |       0.000 |      0.003 |       NA    | Unspecified | NA         |
| Vcco_ddr  |       1.500 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco_mio0 |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vcco_mio1 |       1.800 |     0.000 |       0.000 |      0.000 |       NA    | Unspecified | NA         |
| Vccadc    |       1.800 |     0.020 |       0.000 |      0.020 |       NA    | Unspecified | NA         |
+-----------+-------------+-----------+-------------+------------+-------------+-------------+------------+


1.3 Confidence Level
--------------------

+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+
| User Input Data             | Confidence | Details                                        | Action                                                                                                     |
+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+
| Design implementation state | High       | Design is routed                               |                                                                                                            |
| Clock nodes activity        | High       | User specified more than 95% of clocks         |                                                                                                            |
| I/O nodes activity          | High       | User specified more than 95% of inputs         |                                                                                                            |
| Internal nodes activity     | Medium     | User specified less than 25% of internal nodes | Provide missing internal nodes activity with simulation results or by editing the "By Resource Type" views |
| Device models               | High       | Device models are Production                   |                                                                                                            |
|                             |            |                                                |                                                                                                            |
| Overall confidence level    | Medium     |                                                |                                                                                                            |
+-----------------------------+------------+------------------------------------------------+------------------------------------------------------------------------------------------------------------+


2. Settings
-----------

2.1 Environment
---------------

+-----------------------+------------------------+
| Ambient Temp (C)      | 25.0                   |
| ThetaJA (C/W)         | 11.5                   |
| Airflow (LFM)         | 250                    |
| Heat Sink             | none                   |
| ThetaSA (C/W)         | 0.0                    |
| Board Selection       | medium (10"x10")       |
| # of Board Layers     | 8to11 (8 to 11 Layers) |
| Board Temperature (C) | 25.0                   |
+-----------------------+------------------------+


2.2 Clock Constraints
---------------------

+-------+--------+-----------------+
| Clock | Domain | Constraint (ns) |
+-------+--------+-----------------+
| clk   | clk    |             9.0 |
+-------+--------+-----------------+


3. Detailed Reports
-------------------

3.1 By Hierarchy
----------------

+-------------------------------+-----------+
| Name                          | Power (W) |
+-------------------------------+-----------+
| torus_credit                  |    32.376 |
|   ys[0].xs[0].cli             |     0.047 |
|     regulator                 |     0.041 |
|   ys[0].xs[0].switch          |     2.458 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.076 |
|       gen_vc_logic[0].vc_fifo |     0.072 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[0].xs[1].cli             |     0.033 |
|     regulator                 |     0.023 |
|   ys[0].xs[1].switch          |     2.256 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.092 |
|       gen_vc_logic[0].vc_fifo |     0.087 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[0].xs[2].cli             |     0.032 |
|     regulator                 |     0.022 |
|   ys[0].xs[2].switch          |     2.286 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.115 |
|       gen_vc_logic[0].vc_fifo |     0.110 |
|       gen_vc_logic[1].vc_fifo |     0.002 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[0].xs[3].cli             |     0.056 |
|     regulator                 |     0.046 |
|   ys[0].xs[3].switch          |     2.301 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.138 |
|       gen_vc_logic[0].vc_fifo |     0.134 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[1].xs[0].cli             |     0.042 |
|     regulator                 |     0.024 |
|   ys[1].xs[0].switch          |     1.576 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.131 |
|       gen_vc_logic[0].vc_fifo |     0.126 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[1].xs[1].cli             |     0.056 |
|     regulator                 |     0.048 |
|   ys[1].xs[1].switch          |     1.488 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.075 |
|       gen_vc_logic[0].vc_fifo |     0.071 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[1].xs[2].cli             |     0.041 |
|     regulator                 |     0.031 |
|   ys[1].xs[2].switch          |     1.477 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.102 |
|       gen_vc_logic[0].vc_fifo |     0.098 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[1].xs[3].cli             |     0.034 |
|     regulator                 |     0.026 |
|   ys[1].xs[3].switch          |     1.518 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.093 |
|       gen_vc_logic[0].vc_fifo |     0.089 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[2].xs[0].cli             |     0.034 |
|     regulator                 |     0.024 |
|   ys[2].xs[0].switch          |     2.479 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.127 |
|       gen_vc_logic[0].vc_fifo |     0.123 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[2].xs[1].cli             |     0.053 |
|     regulator                 |     0.042 |
|   ys[2].xs[1].switch          |     2.493 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.100 |
|       gen_vc_logic[0].vc_fifo |     0.096 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[2].xs[2].cli             |     0.035 |
|     regulator                 |     0.023 |
|   ys[2].xs[2].switch          |     2.367 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.083 |
|       gen_vc_logic[0].vc_fifo |     0.079 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[2].xs[3].cli             |     0.050 |
|     regulator                 |     0.040 |
|   ys[2].xs[3].switch          |     2.514 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.102 |
|       gen_vc_logic[0].vc_fifo |     0.098 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[3].xs[0].cli             |     0.034 |
|     regulator                 |     0.024 |
|   ys[3].xs[0].switch          |     1.628 |
|     east_conn_tx              |     0.002 |
|     west_conn_rx              |     0.137 |
|       gen_vc_logic[0].vc_fifo |     0.133 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[3].xs[1].cli             |     0.036 |
|     regulator                 |     0.027 |
|   ys[3].xs[1].switch          |     1.590 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.083 |
|       gen_vc_logic[0].vc_fifo |     0.079 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
|   ys[3].xs[2].cli             |     0.037 |
|     regulator                 |     0.027 |
|   ys[3].xs[2].switch          |     1.615 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.085 |
|       gen_vc_logic[0].vc_fifo |     0.081 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.003 |
|   ys[3].xs[3].cli             |     0.054 |
|     regulator                 |     0.046 |
|   ys[3].xs[3].switch          |     1.654 |
|     east_conn_tx              |     0.001 |
|     west_conn_rx              |     0.117 |
|       gen_vc_logic[0].vc_fifo |     0.113 |
|       gen_vc_logic[1].vc_fifo |     0.001 |
|       gen_vc_logic[2].vc_fifo |     0.002 |
+-------------------------------+-----------+


